src/main/java/io/kestra/storage/minio/MinioStorage.java

```diff
@@ -144,7 +142,7 @@ public Long lastModifiedTime(String tenantId, URI uri) throws IOException {
     @Override
     public FileAttributes getAttributes(String tenantId, URI uri) throws IOException {
         String path = getPath(tenantId, uri);
-        if (!exists(tenantId, uri)) {
+        if (!exists(tenantId, uri) && !path.endsWith("/")) {
```

Comment:
Can you inverse the conditions?
exists() is a costly operation better do it only if needed